sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -20,22 +20,28 @@
 
 SERVICEACCOUNT_NAMESPACE = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
 GCS_STAGING_BLOB_DEFAULT_PREFIX = 'kfp_container_build_staging'
+GCR_DEFAULT_IMAGE_SUFFIX = 'kfp_container'
 
 class ContainerBuilder(object):
   """
   ContainerBuilder helps build a container image
   """
-  def __init__(self, gcs_staging=None, namespace=None):
+  def __init__(self, gcs_staging=None, gcr_image_tag=None, namespace=None):
     """
     Args:
       gcs_staging (str): GCS bucket/blob that can store temporary build files,
           default is gs://PROJECT_ID/kfp_container_build_staging.
+      gcr_image_tag (str): GCR image tag where the target image is pushed
```

Comment:
The `gcr_image_tag` parameter name might be confusing:
1. I'm not sure it's GCR-specific
2. Docker already uses the term `tag` for a different concept

What do you think about naming it `default_image_name`?

Reply:
it currently only support gcr, thus naming it this way. 
I'm OK with renaming it once we support other registries.